src/editors/config/OAuthInput.tsx

```diff
@@ -115,6 +115,18 @@ export const OAuthInputsEditor = (props: DataSourcePluginOptionsEditorProps<Infi
               secureFieldValue="oauth2EndPointParamsValue"
             />
           </div>
+          <div className="gf-form">
```

Comment:
Can we move this into the `Customize OAuth2 token (Advanced)` section? 

Since this is a optional thing, having it in the default settings may confuse the users.